vidarr-server/src/main/java/ca/on/oicr/gsi/vidarr/server/Main.java

```diff
@@ -1087,7 +1087,10 @@ private void createAnalysisRecords(
         .forEach(
             result -> {
               try {
-                jsonGenerator.writeRawValue(result.value1().data());
+                String rawResult = result.value1().data();
+                jsonGenerator.writeRawValue(rawResult.substring(0, rawResult.length() - 1)
+                    + ",\"instanceName\": \"" + selfName + "\"}");
+
```

Comment:
Rather than modify the result after it's run through the database, you could add the `instanceName` field+value to the set of `fields` to be selected - the jOOQ equivalent of psql's `SELECT completed, created, "randomString" AS instanceName, id, modified FROM workflow_run ...` where "randomString" would be selfName